librariejs.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Project for nuget pack</Description>
+    <Authors>Autodesk</Authors>
+    <PackageTags>Dynamo;librariejs</PackageTags>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <versionprefix>1.0.0</versionprefix>
+    <PackProjectInputFile>$(MSBuildProjectFullPath)</PackProjectInputFile>
+    <NoBuild>true</NoBuild>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
```

Comment:
can you change this to .net 4.7 - as that is how we will the project version we will consume this from for now.

Reply:
We can´t, because the docker doesn't have the full net standard framework and causes conflicts on the dotnet packing.

Reply:
okay, we need to make sure that we can actually reference this in Dynamo then. Theres no point in publishing it if Dynamo.sln cannot consume it.

Alternatively instead of using a csproj - you can just use nuget and .nuspec file like this:
https://github.com/DynamoDS/Dynamo/blob/master/tools/NuGet/template-nuget/DynamoVisualProgramming.Core.nuspec

and call nuget directly to package the nuspec into a nupkg

Reply:
This is the error that we got, whit 4.7 version.

![image](https://user-images.githubusercontent.com/48030764/66147113-c6f02d80-e5d3-11e9-82d3-7fdf6512915e.png)

Reply:
We can´t use nuget commands on a docker linux container. We also tried the .nuspec file on several tests.

Reply:
thanks @jesusHCG we might need to go back to the drawing board on this then - can you send me the generated nupkg file - we need to be certain Dynamo.sln can actually consume it - we should not require devs to install dotnetcore 2 just to consume this single package.

If it does not work we'll need to figure something else out like using a windows docker instance.

Reply:
What about using: 
--no-build option when doing the -pack

Reply:
@mjkkirschner The nupkg will work for Dynamo, since we are going to do similar steps as we are doing with the ASM lib, install the package on an specific location and copy the necessary files to the project (min.js and resources)

Reply:
I don't think that is the ideal workflow - I would ideally like the nuget reference added to the librarieJSExtension.csproj file - this way we can easily use it or update it as part of the dynamo.sln in any build job.

Reply:
consider the case that dynamo repo is being built in another build pipeline, github actions, etc or on a users machine (it is open source) etc - 
it's easiest and *best* if we can take advantage of nuget restore within visual studio, not need to update some config file somewhere to change the reference to a new version of librariejs.

Reply:
@mjkkirschner I have a question there, if the plan is to keep it up inside .sln, so anyone can build and take advantage of the restore, isn't going to generate an error since the .nupkg will be available only on ADSK artifactory? Or are we going to publish it to nuget.org?

Reply:
published to nuget - dynamo always needs to build for people outside ADSK.

Reply:
Ok, since the ideal workflow is totally different from the one that we agree before on the standups, we need to redirect the efforts to a different approach/solution